extension/js/common/core/crypto/pgp/openpgp-key.ts

```diff
@@ -269,6 +269,7 @@ export class OpenPGPKey {
         throw new Error(`Key ${key.id} is not a private key`);
       }
       opgpKey = (await opgp.revokeKey({ key: opgpKey, format: 'object' })).privateKey;
+      OpenPGPKey.patch(opgpKey);
```

Comment:
Perhaps, it makes sense to call ``OpenPGP.patch` each time a new OpenPGP.Key object is created (by reading / fixing / generating etc.), and we also need to check message objects